testsuite/features/support/commonlib.rb

```diff
@@ -306,7 +306,7 @@ def transactional_system?(name, runs_in_container: true)
 # @return [Boolean] true if the host name belongs to a Red Hat-like distribution, false otherwise
 def rh_host?(name)
   os_family = get_target(name).os_family
-  %w[rocky centos redhat alma oracle liberty].include? os_family
+  %w[rocky centos redhat alma oracle liberty almalinux ol rhel].include? os_family
```

Comment:
That's weird as I had been checking the different os family shown on the salt grains, but I might do it wrong. Approving.